dev-python/ipywidgets/ipywidgets-9999.ebuild

```diff
@@ -29,6 +29,7 @@ RDEPEND="
 	"
 DEPEND="${RDEPEND}
 	test? (
+		$(python_gen_cond_dep 'dev-python/mock[${PYTHON_USEDEP}]' python2_7)
```

Comment:
Right, it is only for 2.7 https://github.com/ipython/ipywidgets/blob/master/setup.py#L286, all of the IPython packages that use mock do a try/except ImportError on unittest.mock before importing from mock (https://github.com/ipython/ipywidgets/blob/master/ipywidgets/widgets/tests/test_interaction.py#L8)